curriculum/challenges/english/blocks/lecture-working-with-arrays-variables-and-naming-practices/6732c6ba2ea42b610b9f9ce1.md

```diff
@@ -20,57 +20,41 @@ console.log(fruits.length); // 3
 
 It's possible to have arrays with empty slots. Empty slots are defined as slots with nothing in them. This is different than an array with the value of `undefined`. These types of arrays are known as sparse arrays. Here is an example:
 
-:::interactive_editor
-
 ```js
 const sparseArray = [1, , , 4];
 console.log(sparseArray.length); // 4
 ```
 
-:::
-
-In this case even though we only have two defined elements, `1` and `4`. The length is `4` because the highest index (`3`) plus `1` gives us a length of `4`.
+In this case even though we only have two defined elements, `1` and `4`, the length is `4` because the highest index (`3`) plus `1` gives us a length of `4`. The empty slots in the middle do not contain `undefined`; they are truly empty.
```

Comment:
I don't think we should add this last sentence because it will be confusing for beginners. If they are lookin at the code, there is no mention of undefined. So mentioning it here will be confusing.

```suggestion
In this case even though we only have two defined elements, `1` and `4`, the length is `4` because the highest index (`3`) plus `1` gives us a length of `4`. 
```